src/niscope/templates/_grpc_stub_interpreter.py/fetch_array_measurement.py.mako

```diff
@@ -0,0 +1,7 @@
+
+    def fetch_array_measurement(self, channel_list, timeout, array_meas_function, measurement_waveform_size):  # noqa: N802
```

Comment:
Note: I still needed to add this template to not forward the `measurement_waveform_size` argument on to the gRPC request.